eslint.config.mjs

```diff
@@ -0,0 +1,38 @@
+import globals from "globals";
+import pluginJs from "@eslint/js";
+import tseslint from "typescript-eslint";
+import eslintConfigPrettier from "eslint-config-prettier";
+
+export default [
+  { files: ["**/*.{js,mjs,cjs,ts}"] },
+  { ignores: ["**/.gradle", "**/dist/", "**/build/"] },
+  {
+    languageOptions: {
+      ecmaVersion: "latest",
```

Comment:
Is that safe? Or might we get spurious linting error when "latest" increments?

Reply:
I just applied the value we had before.
I have to think about what effect specifying `[ecmaVersion](https://typescript-eslint.io/packages/parser/#ecmaversion)` would have.

Reply:
I do not fully understand what it does.
For me, it is out of scope to look into what effects changing the `ecmaVersion` has and to reevaluate our previous configuration of it.
Therefore, I want to keep it as it was in this PR.